readthedocs/doc_builder/environments.py

```diff
@@ -910,7 +911,8 @@ def get_container_name(self):
                 project_name=self.project.slug,
             )
         else:
-            name = f'sync-project-{self.project.pk}-{self.project.slug}'
+            uuid = uuid.uuid4()
```

Comment:
Also needs a comment explaining why this is here. Something like "This avoids hitting a docker error when we try and run two of the same tasks at the same time"